src/app/src/Map/Map.tsx

```diff
@@ -1,24 +1,32 @@
 import './Map.css'
 import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet'
 import 'leaflet/dist/leaflet.css'
+import Button from 'react-bootstrap/Button';
 
 
 function Map() {
 
   return (
-    <div className="leaflet-container">    
-      <MapContainer center={[39.952584, -75.165222]} zoom={13} scrollWheelZoom={true}>
-          <TileLayer
-            attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
-            url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-          />
-          <Marker position={[39.952584, -75.165222]}>
-            <Popup>
-              This spots got a good vibe
-            </Popup>
-          </Marker>
-      </MapContainer>
+    <div className="leaflet-container-wrapper">
+      <div className="leaflet-container">    
+        <MapContainer center={[39.952584, -75.165222]} zoom={13} scrollWheelZoom={true}>
+            <TileLayer
+              attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+              url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+            />
+            <Marker position={[39.952584, -75.165222]}>
+              <Popup>
+                This spots got a good vibe
+              </Popup>
+            </Marker>
+        </MapContainer>
       </div>
+      <div className="button-container">
+        <Button style={{backgroundColor:"#ADD8E6", marginLeft: '1rem', marginRight: '1rem'}} variant="primary">Near Me</Button>
```

Comment:
The objects in the style section are all the same and could be refactored out to a `const` variable and used in each of the style props.